### ISSUE TITLE
map-reduce.co is down

### ISSUE THREAD

**Issue by umaar** [remy/map-reduce#1]
Thanks for the cool service!

https://map-reduce.co/ is down right now, don't suppose you have any plans to bring it up?

**Comment by remy** [comment]
Yeah, I parked the domain for a bit partly because I didn't have domain availability on Zeit's now (without going into PAYG), and partly because I wasn't sure this was a good idea or not!

I'm now wondering…at 6am…whether the way to solve this is to put the functions on AWS S3 and sync them down on startup/deploy.

Looking through the code is tricky too since I did a massive copy from https://github.com/remy/jsonbin!

Did you see yourself having a need for it? If so, I can see if I can get it up and running again, perhaps without the need for a mongodb too.

**Comment by umaar** [comment]
Very kind of you to offer, actually I was only interested in playing around with it, didn't have a hard use case or anything. Also I wonder, if the functions are just code, maybe they could be stored as gists or something!

**Comment by remy** [comment]
They're stored as "sort of code". I'll put up a few examples tomorrow. The
big reason (now that I revisited) that I didn't leave it live was because
the security hardening would have been a lot more work. Ideally the code
would run in a sandbox environment which is possible, but a pain to
re-enforce.
